ChernobaiYaroslav/HW6/task1.py

```diff
@@ -0,0 +1,34 @@
+def lineReturnFromList(numsList: list[int])->str:
+    """Return a string from a list consisting of integer values"""
+    return " ".join(map(str, numsList))
+
+###############
+# Solution №1
+###############
+
```

Comment:
output should be the next
[2, 4, 6, 8, 10] is even multiple of 2
[3, 9] is an odd multiple of 3
[1, 5, 7] not divisible by 2 and 3

look at this version of code
l1 = [ x for x in range(1, 11) if x % 2 == 0 ]
l2 = [ x for x in range(1, 11) if x % 2 == 1 and x % 3 == 0 ]
l3 = [ x for x in range(1, 11) if not x % 2 == 0 and not x % 3 == 0 ]
print(l1, 'is even multiple of 2')
print(l2, 'is an odd multiple of 3')
print(l3, 'not divisible by 2 and 3')

or


even = []
odd = []
other = []

for number in range(1, 11):
    if number >= 2 and number % 2 == 0:
        even.append(number)
    elif number >= 3 and number % 3 == 0:
        odd.append(number)
    else:
        other.append(number)

print(f"Even numbers that are divisible by 2: {even}")
print(f"\nOdd numbers, which are divisible by 3: {odd}")
print(f"\nNumbers that are not divisible by 2 and 3: {other}")